.evergreen/run-tests.sh

```diff
@@ -21,6 +22,12 @@ export PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig"
 # override where we look for libmongoc
 export LD_LIBRARY_PATH="${INSTALL_DIR}/lib"
 export DYLD_LIBRARY_PATH="${INSTALL_DIR}/lib"
+export DEVELOPER_DIR=/Applications/Xcode10.1.app
 
 swiftenv local $SWIFT_VERSION
-MONGODB_TOPOLOGY=${TOPOLOGY} MONGODB_URI=$MONGODB_URI make test
+
+# build the driver
+swift build $EXTRA_FLAGS
```

Comment:
[nit] `swift test` will build the driver before running tests, so this isn't exactly necessary. If you purposefully want a separate stage for building then this is fine, though.

Reply:
I considered calling just `swift test`, but wanted to make the two calls explicit, especially since only one of the calls requires passing additional environment variables in.